deploy/role.yaml

```diff
@@ -0,0 +1,38 @@
+# auxiliary permissions needed to run the operator
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: osd-metrics-exporter
+  namespace: openshift-osd-metrics
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - pods
+      - services
+      - endpoints
+      - events
+      - secrets
+    verbs:
+      - "*"
+  - apiGroups:
+      - ""
+    resources:
+      - namespaces
```

Comment:
This is just to get the local namespace right?

Reply:
Yes, exactly. But i'm wondering what this permission is actually used for now that you ask about it. Removing it doesn't seem to impact the operator, but i'm unsure how this would  be different when the operator is to be deployed with OLM instead of locally.